bitnami/external-dns/templates/clusterrole.yaml

```diff
@@ -100,6 +101,7 @@ rules:
       - tlsroutes
       - tcproutes
       - udproutes
+      - grpcroutes
```

Comment:
(Interested party chiming in to speed up approval:) The existing items are in alphabetical order.